pyca/ingest.py

```diff
@@ -139,6 +140,10 @@ def control_loop():
                              .filter(RecordedEvent.status ==
                                      Status.FINISHED_RECORDING).first()
         if event:
+            delay = random.randint(config()['ingest']['delay_min'],config()['ingest']['delay_max'])
```

Comment:
I just realized that you could also use the arguably nicer form of:
```suggestion
            delay = random.randint(config('ingest', 'delay_min'), config('ingest', 'delay_max'))
```

Reply:
thank you. changes commited